core/controllers/release_coordinator.py

```diff
@@ -142,63 +138,39 @@ def get(self) -> None:
         })
 
     @acl_decorators.can_access_release_coordinator_page
-    def delete(self) -> None:
-        """Performs deletion on the specified user group."""
+    def post(self) -> None:
+        """Performs series of action based on action parameter for user
+        groups.
+        """
         assert self.user_id is not None
```

Comment:
Does this not already happen in the can_access_release_coordinator_page decorator?

Reply:
Ah, right. I have updated it.